### ISSUE TITLE
Webview blocks loading files from extensions

### ISSUE THREAD

**Issue by azer** [nwjs/nw.js#5962]
If I load a page using `iframe` and `nwfaketop`, extensions get loaded and work properly. Using webview fails though, I get `ERR_BLOCKED_BY_CLIENT` errors for loading files from extensions. The error message doesn't have any helpful information.

I could use iframes instead of webviews to avoid this problem but iframes doesn't let me control the cookies. If they do, I'd like to ask how :)

**Comment by Christywl** [comment]
Could you please provide a sample to reproduce it?

**Comment by rogerwang** [comment]
Yeah, would like to support it after knowing what's happening with a sample.

btw, in iframe scenario, the chrome.cookies API should work: https://developer.chrome.com/extensions/cookies

**Comment by azer** [comment]
I'll see if I can prepare a sample but it'd probably be difficult. @rogerwang how would cookies API be used for resetting cookies of a specific iframe ? In my case, I have multiple iframes running same app, and each needs to have separate cookies.

**Comment by frankhale** [comment]
Similar related issue: https://github.com/nwjs/nw.js/issues/4878

**Comment by frankhale** [comment]
Here is a simple example to demonstrate the issue: https://github.com/frankhale/nwjs-react-devtools-webview-example

**Comment by Christywl** [comment]
Using frankhale's example, I can reproduce this issue on Linux/Windows with nwjs-sdk-v0.23.6.
![image](https://user-images.githubusercontent.com/20039821/28296532-651e50dc-6b9b-11e7-9f86-88102a521a8e.png)
